### PR TITLE
Added SNS script for Golang

### PR DESCRIPTION
**Pull request by kschumy**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by kschumy** [comment]
/hold

Confirming account permissions are correct

**Comment by kschumy** [comment]
/unhold

ready for review

File: projects/golang/go/scripts/eks_golang_release_notification.sh [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    echo "Empty SNS_TOPIC_ARN"
+    exit 1
+fi
+
+if [ "$GO_SOURCE_VERSION" == "" ]; then
+    echo "Empty GO_SOURCE_VERSION"
+    exit 1
+fi
+
+base_directory=$(git rev-parse --show-toplevel)
```
Comment (rcrozean) [inline projects/golang/go/scripts/eks_golang_release_notification.sh:26]: Just a nit, capitalize the variable names. BASE_DIRECTORY

File: projects/golang/go/scripts/debian_image_release_notification.sh [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    echo "Empty SNS_TOPIC_ARN"
+    exit 1
+fi
+
+if [ "$GO_SOURCE_VERSION" == "" ]; then
+    echo "Empty GO_SOURCE_VERSION"
+    exit 1
+fi
+
+base_directory=$(git rev-parse --show-toplevel)
```
Comment (rcrozean) [inline projects/golang/go/scripts/debian_image_release_notification.sh:26]: Same nit, capitalize variable names

Comment (kschumy) [reply projects/golang/go/scripts/debian_image_release_notification.sh]: done

**Pull request merged**